src/main/java/chess/domain/position/File.java

```diff
@@ -0,0 +1,79 @@
+package chess.domain.position;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toMap;
+
+public class File {
+    private static final Map<String, File> CACHE = IntStream.rangeClosed('a', 'h')
+                                                            .mapToObj(i -> String.valueOf((char) i))
+                                                            .collect(toMap(Function.identity(), File::new));
```

Comment:
```suggestion
            .mapToObj(i -> String.valueOf((char) i))
            .collect(toMap(Function.identity(), File::new));
```
intellij code style이 어떻게 적용되어있을까요? `.`을 이렇게 나란히 사용하지는 않아요.

Reply:
![image](https://github.com/woowacourse/java-chess/assets/125939503/84233585-a2b1-4418-a534-857e22042b85)

크루의 추천을 받고 해당 옵션을 설정해뒀었는데 쓰이지 않는 옵션인가보군요...
깔끔해 보여서 좋았는데 사용하지 않는다니 너무 아쉽습니다..😭
바로 수정하였습니다!!